modules/nodes-pods-priority-about.adoc

```diff
@@ -37,6 +38,8 @@ system-node-critical      2000001000   false            72m
 ** sdn-ovs
 ** sync
 
+* *openshift-user-critical* - Prometheus pods under `openshift-monitoring` namespace use `priorityClassName` `openshift-user-critical`. The monitoring workloads use `system-critical` as their first `priorityClass`.But it causes problem when monitoring uses excessive memory and the nodes can't evict them. Monitoring priority drops to give the scheduler flexibility, to move the heavy workloads around and keep critical nodes alive.
+
 * *system-cluster-critical* - This priority class has a value of 2000000000 (two billion) and is used with pods that are important for the cluster. Pods with this priority class can be evicted from a node in certain circumstances. For example, pods configured with the `system-node-critical` priority class can take priority. However, this priority class does ensure guaranteed scheduling. Examples of pods that can have this priority class are fluentd, add-on components like descheduler, and so forth.
 A number of critical components include the `system-cluster-critical` priority class by default, for example:
 +
```

Comment:
Better organize the Priority cluster according to the values, descending order

Reply:
acked